### PR TITLE
Added support for new field custom cron

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
This is related to Satellite 6.5 Feature to support custom_cron. 

- Added this Create View Page 
- Same also added in Edit View Page with Recurring Logic 

Corresponding tests will be added in robottelo soon.

### CODE REVIEW

File: airgun/views/syncplan.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ class SyncPlanCreateView(BaseLoggedInView):
     name = TextInput(id='name')
     description = TextInput(id='description')
     interval = Select(id='interval')
+    cron_expression = TextInput(id='cron_expression')
```
Comment (oshtaier) [inline airgun/views/syncplan.py]: that should be implemented using conditional view, let's me check that

Comment (oshtaier) [reply airgun/views/syncplan.py]: okay, that will probably add unnecessary complexity

File: airgun/views/syncplan.py [hunk 2]
```diff
@@ -73,7 +74,9 @@ class details(SatTab):
         next_sync = ReadOnlyEntry(name='Next Sync')
         enabled = EditableEntryCheckbox(name='Sync Enabled')
         interval = EditableEntrySelect(name='Interval')
+        cron_expression = EditableEntry(name='Cron Logic')
```
Comment (oshtaier) [inline airgun/views/syncplan.py]: just change order to have fields in the same way we have them on UI:
```
name = EditableEntry(name='Name')
description = EditableEntry(name='Description')
date_time = EditableDateTime(name='Start Date')
next_sync = ReadOnlyEntry(name='Next Sync')
recurring_logic = ReadOnlyEntry(name='Recurring Logic')
enabled = EditableEntryCheckbox(name='Sync Enabled')
interval = EditableEntrySelect(name='Interval')
cron_expression = EditableEntry(name='Cron Logic')
products_count = ReadOnlyEntry(name='Products')
```

Comment (omkarkhatavkar) [reply airgun/views/syncplan.py]: Done!

**Pull request merged**